docs/community/README.md

```diff
@@ -1,2 +1,21 @@
-# `jupyter` accessibility community information
+# `jupyter` accessibility community
+
+Once we have assembled the working group, we will list the members here.
+
+## Join our JupyterLab accessibility meetings
+
+Anyone is welcome to attend, if they would like to discuss a topic or just to listen in.
+
+- When: every other Wednesday (check the [community calendar](https://jupyter.readthedocs.io/en/latest/community/content-community.html#jupyter-community-meetings)) at [10:15 AM Pacific Time](https://dateful.com/convert/san-francisco-california?t=1015am)
+- Where: [`jovyan` Zoom](https://zoom.us/my/jovyan?pwd=c0JZTHlNdS9Sek9vdzR3aTJ4SzFTQT09)
+- What: discussions on current work and feedback
+
+### Meeting notes and reports
+
+- [current agenda](https://hackmd.io/WnaWXboXSiGoqWvev_fAvA)
+- [public notes archive](https://github.com/jupyterlab/team-compass/issues/98) from past meetings
```

Comment:
```suggestion
- [public notes archive](docs/community/meeting-minutes/jupyterlab-accessibility-meetings/all-minutes.md)
```

Reply:
i am linking to the all minutes markdown on line 18. Do we just not want the old archive at all anymore? or does the "all meeting minutes" include the "issues/98" meeting minutes too?
```
- [All JupyterLab Accessibility Meeting Minutes](./meeting-minutes/jupyterlab-accessibility-meetings/all-minutes.md)
```

Reply:
`all-minutes.md` does have everything listed on jupyterlab/team-compass #98. I'd prefer to point people in one place, but I won't block it if you see benefit in keeping the old link too.